### DISCUSSION TITLE
getting DC error after 20secs delay between frontend and backend pod

### DISCUSSION THREAD

**Discussion by parshva-shah-dsw** [istio/istio#53817] [General]
we are getting below logs seen in the istio proxy side car of frontend pods

log: 2024-09-27T12:25:11.984388848Z stdout F {"upstream_service_time":null,"start_time":"2024-09-27T12:24:51.891Z","method":"POST","user_agent":"Apache-HttpClient/4.5.13 (Java/11.0.23)","bytes_sent":0,"response_flags":"DC","upstream_local_address":"0000","route_name":null,"bytes_received":75,"requested_server_name":null,"authority":"fnhttp-py-svc","path":"/getUsr","upstream_host":"0000","response_code":0,"protocol":"HTTP/1.1","request_id":"xxx","duration":20016,"x_forwarded_for":null,"upstream_cluster":"outbound|svc","downstream_local_address":"000","connection_termination_details":null,"response_code_details":"downstream_remote_disconnect","upstream_transport_failure_reason":null,"downstream_remote_address":"0000"}
}
can you help us with any resolution as this issue is seen very intermittently and there is no specific pattern of occurrences?

istio 1.23.0
EKS kubernetes 1.29.5
aws load balancer

**Comment by zirain** [comment]
maybe there's some node/pod have network issue between them because of firewall or security group.

**Reply by parshva-shah-dsw** [reply]
everything is in a single private subnet
